exonum-java-binding/testkit/src/test/java/com/exonum/binding/testkit/TestKitTestWithArtifactsCreated.java

```diff
@@ -56,8 +56,14 @@ private static void createTestServiceArtifact() throws IOException {
 
   private static void createTestService2Artifact() throws IOException {
     Path artifactLocation = artifactsDirectory.resolve(ARTIFACT_FILENAME_2);
-    createArtifact(artifactLocation, ARTIFACT_ID_2, TestServiceModule2.class, TestTransaction.class,
-        TestSchema.class, TestService2.class);
+    createArtifact(artifactLocation, ARTIFACT_ID_2, TestServiceModule2.class,
+        TestTransaction.class, TestSchema.class, TestService2.class);
+  }
+
+  static void createInvalidArtifact(String filename) throws IOException {
```

Comment:
We again share the temporary directory — why? Inject a separate one into the test that needs it.